ansible.cfg

```diff
@@ -1,7 +1,7 @@
 [defaults]
 nocows                  = 1
 # Don't forget to update other ansible.cfg files like:  ansible/configs/linklight/ansible.cfg
-roles_path              = dynamic-roles:ansible/dynamic-roles:roles-infra:ansible/roles-infra:roles:ansible/roles:ansible/roles-studentvm:roles-studentvm
+roles_path              = dynamic-roles:ansible/dynamic-roles:roles-infra:ansible/roles-infra:roles:ansible/roles:ansible/roles_studentvm:roles_studentvm
```

Comment:
Did we want to drop the `ansible/` as @fridim suggested or leaving it for now?

Reply:
I'll let @fridim do that. :-)